tox.ini

```diff
@@ -35,6 +35,18 @@ docker = healthcheck/redis:alpine
 deps = pytest
 commands = py.test [] test_healthcheck.py
 
+[testenv:healthcheck-arbitrary]
+docker = redis:5-alpine3.11
```

Comment:
how is this different from `healthcheck-builtin`? I see the image is different, and may have a different builtin healthcheck, but does it exercise different behavior of tox-docker? If not, let's remove one or the other to keep the test suite smaller than larger...

Reply:
It is different because of the healthcheck I added.  I had to add a new image in order to add a different configuration or I got a duplicate configuration error.

```
healthcheck_cmd = false || true
```

I chose this because of the current parsing problem.  It appears that the first "word" is used as the command in master (before this patch is applied).  If that happens here, the `false` command will fail.  It will also fail if it is parsed such that `false` is run and "|| true" is passed as a string argument.  The only way for it to pass is if the healthcheck runs the entire "false || true" through the shell.

I was hesitant to remove a test already in the library.  I can remove the original one, but I would keep the "false || true" test.

Reply:
Ah, of course. Let's get rid of the old one, or rather, just use `false || true` as the healthcheck for the `healthcheck-custom` env.

Can you check whether quoted health checks (as I have at https://gitlab.com/bigapplepy/yak-bak/-/blob/master/tox.ini#L63) work with these changes? If not, this'll be a backwards-breaking change, and will need to go into tox-docker 2.0 (along with a bunch of other things I've wanted to berak for a while, but hadn't needed to yet).

Reply:
I'll include this in 2.0, along with the other changes... I think they should be relatively quick to get done.